Makefile

```diff
@@ -70,7 +70,7 @@ libbpfgo-static: $(VMLINUXH) | $(LIBBPF_OBJ)
 		CGO_CFLAGS=$(CGO_CFLAGS_STATIC) \
 		CGO_LDFLAGS=$(CGO_LDFLAGS_STATIC) \
 		GOOS=linux GOARCH=$(ARCH) \
```

Comment:
This change breaks the go build on x86 because the correct architecture argument for GOARCH is `amd64`. 

https://github.com/golang/go/blob/master/src/go/build/syslist.go#L52

Reply:
I am confused, I thought that it should work
```
$ echo x86_64 | sed 's/x86_64/amd64/g; s/aarch64/arm64/g'
amd64
```

What's the error you are getting?

Reply:
`cmd/go: unsupported GOOS/GOARCH pair linux/x86`. in `selftest/common/Makefile` youre replacing `x86_64` with `x86`, not `amd64`

Reply:
Oh! My bad. Was not looking at the right file!! 🤦